[RHACS] Post-GA 3.72 Change Requests (#50942)
openshift/openshift-docs: modules/view-nonactive-clusters.adoc

// Module included in the following assemblies:
//
// * configuration/cluster-decommissioning.adoc
:_content-type: PROCEDURE
[id="view-nonactive-clusters_{context}"]
= Viewing nonactive clusters

> 

Reviewer: You need an abstract here.

.Procedure
. On the {product-title-short} portal, navigate to *Platform Configuration* -> *System Configuration*.
. In the *Cluster deletion* section, click *Clusters which have Sensor Status: Unhealthy* to go to the *Clusters* list page. This page is filtered to show nonactive clusters that are subject to removal from {product-title-short} and a timeframe for the removal. 
+
[NOTE]
====

> If this feature is enabled after a cluster is considered nonactive, the counting of days towards removal starts from the time the cluster became nonactive, not from the time that the feature was enabled. If there are any nonactive clusters that you do not want removed, you can configure a label as described in the "Configuring cluster decommissioning" section. Labeled clusters are ignored when the system removes nonactive clusters.

Reviewer 2: ```suggestion
If this feature is enabled after a cluster is considered nonactive, the counting of days towards removal starts from the time the cluster became nonactive, not from the time that the feature was enabled. If there are any nonactive clusters that you do not want removed, you can configure a label as described in the "Configuring cluster decommissioning" section. Clusters with those labels are not removed even if they are nonactive.
```

Something like that? The current wording made me think all labelled clusters are not removed which is incorrect. Only the ones with a label that are specified.

// Module included in the following assemblies:
//
// * configuration/cluster-decommissioning.adoc
:_content-type: PROCEDURE
[id="view-nonactive-clusters_{context}"]
= Viewing nonactive clusters

[role="_abstract"]

> Nonactive clusters are clusters that were installed and performed a handshake with Central at least once, but have been unreachable by Sensor for a specified period of time. Use this procedure to view a list of these clusters.

Reviewer: ```suggestion
Nonactive clusters are clusters that were installed and performed a handshake with Central at least once but have been unreachable by Sensor for a specified period of time. Use this procedure to view a list of these clusters.
```
